public/silent-renew.html

```diff
@@ -0,0 +1,7 @@
+<script src="https://cdnjs.cloudflare.com/ajax/libs/oidc-client/1.5.1/oidc-client.js"></script>
```

Comment:
Don't use a cdn, we are hosting this ourselves

Reply:
Don't add another html file in public folder, do this in our app using react router